### PR TITLE
fix typo, this let's -> this lets

### PR DESCRIPTION
**Pull request by fancyuan**
**What this PR does / why we need it**:
fix typo

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: plugin/pkg/admission/resourcequota/resource_access.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ type quotaAccessor struct {
 	lister corelisters.ResourceQuotaLister
 
 	// liveLookups holds the last few live lookups we've done to help ammortize cost on repeated lookup failures.
-	// This let's us handle the case of latent caches, by looking up actual results for a namespace on cache miss/no results.
+	// This let us handle the case of latent caches, by looking up actual results for a namespace on cache miss/no results.
```
Comment (nikhita) [inline plugin/pkg/admission/resourcequota/resource_access.go]: s/let/lets

File: plugin/pkg/admission/resourcequota/resource_access.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ type quotaAccessor struct {
 	lister corelisters.ResourceQuotaLister
 
 	// liveLookups holds the last few live lookups we've done to help ammortize cost on repeated lookup failures.
-	// This let's us handle the case of latent caches, by looking up actual results for a namespace on cache miss/no results.
+	// This lets us handle the case of latent caches, by looking up actual results for a namespace on cache miss/no results.
```
Comment (hzxuzhonghu) [inline plugin/pkg/admission/resourcequota/resource_access.go]: let's or let us

Comment (CaoShuFeng) [reply plugin/pkg/admission/resourcequota/resource_access.go]: > let's or let us

Haha, I can't aggre. `This lets us` is right.

Comment (CaoShuFeng) [reply plugin/pkg/admission/resourcequota/resource_access.go]: /ok-to-test

**Comment by nikhita** [comment]
@fancyuan Can you squash the commits to a single commit?

**Comment by derekwaynecarr** [comment]
thank you for the cleanup.
/approve
/lgtm

**Pull request merged**